tests/clients_last_seen_v1/test_single_day/clients_daily_v6.schema.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "submission_date_s3",
+    "type": "DATE",
+    "mode": "REQUIRED",
+    "description": "time_partitioning_field"
```

Comment:
RFC on using this to locate `time_partitioning_field` and perhaps `clustering_field`s or moving it to a separate file, which could be merged with `query_params.yaml` into a `test_info.yaml` or something? that would prevent the proliferation of small files for additional test info and in small enough cases could be an appropriate place for all of the data for a test.

Reply:
This is indeed looking like quite a bit of configuration to manage for writing a test, so making some optimizations for easier test writing is appropriate. Putting time_partitioning_field in description seems reasonable and self-documenting.

Reply:
okay, i'll leave it here for now, and in a follow-up PR add support for reading `tests/clients_last_seen_v1/test_*.yaml` and `tests/clients_last_seen_v1/test_*/test.yaml` as a place to put things, which should also enable `time_partitioning_field` with `autodetect`